tests/regression/lp-1898038/test-multipass-support-app/meta/snap.yaml

```diff
@@ -0,0 +1,9 @@
+name: test-multipass-support-app
+version: 1
+
+apps:
+    test-multipass-support:
+        daemon: simple
```

Comment:
It is usually easier to run apps directly, unless testing service behavior is required. This avoids systemd restarting conditions and related complexity, and allows one to easily measure the outcome and error status.

Reply:
Thanks - I just added this in abf0f62